__tests__/__unit__/utils/workspace.unit.test.ts

```diff
@@ -0,0 +1,157 @@
+import * as vscode from "vscode";
+import { closeOpenedTextFile, checkTextFileIsOpened } from "../../../src/utils/workspace";
+
+jest.mock("vscode");
+
+const activeTextEditor = jest.fn();
+const executeCommand = jest.fn();
+
+Object.defineProperty(vscode.window, "activeTextEditor", {get: activeTextEditor});
+Object.defineProperty(vscode.commands, "executeCommand", {value: executeCommand});
+
+describe("Checking Workspace file status method", () => {
+    beforeEach(() => {
+        activeTextEditor.mockReset();
+        executeCommand.mockReset();
+    });
+
+    it("Tests if method runs correctly with no tabs available", async () => {
```

Comment:
```suggestion
    it("Runs correctly with no tabs available", async () => {
```
and similarly for other test titles ...

Reply:
Sounds logical.